packages/atclient/CMakeLists.txt

```diff
@@ -146,6 +147,24 @@ if(NOT ESP_PLATFORM)
 	add_subdirectory(${CJSON_DIR})
 	list(APPEND ATCLIENT_TARGETS_TO_INSTALL cjson-static)
 
+	# #########################################################
+  # 5D. Create atlogger library
+  # - atlogger header files are installed to `/usr/local/include/atlogger`
+  # - atlogger library is automatically linked as part of atclient library (when you install atclient, it already includes atlogger)
+  # - atlogger library is only exported as a standalone library if ATCLIENT_EXPORT_ATLOGGER is set to ON
+	# #########################################################
+  if(ATCLIENT_EXPORT_ATLOGGER)
+    add_library(atlogger STATIC ${CMAKE_CURRENT_LIST_DIR}/src/atlogger.c)
+    add_library(${PROJECT_NAME}::atlogger ALIAS atlogger)
```

Comment:
Q for you @JeremyTubongbanua should the standalone alias be atclient::atlogger or atlogger::atlogger? I guess if we have to make it its own package later, it will have to be atlogger::atlogger anyway...